packages/playground-server/src/db.ts

```diff
@@ -7,16 +7,16 @@ import { v4 as generateUuid } from "uuid";
 import Errors from "./errors";
 import User, { MatchedUser } from "./resources/user/resource";
 
-const DATABASE_URL = (process.env.DATABASE_URL as string) || (process.env.DB_CONNECTION_STRING as string);
+const DATABASE_URL =
+  (process.env.DATABASE_URL as string) ||
+  (process.env.DB_CONNECTION_STRING as string);
```

Comment:
Are these env variables documented anywhere? e.g., in the README
Also kind of confused why there are two for the same thing?

Reply:
@snario I think it's for @ebryn's experiment of deploying to Heroku. We will probably later unify and have a single variable for this. There's a mention in the `README`, but definitely needs improvement.